build.gradle

```diff
@@ -518,3 +518,12 @@ subprojects {
     }
   }
 }
+
+subprojects {
+  Closure disableMethod = { String... tasknames ->
+    for (String taskname : tasknames) {
+      project.tasks.named(taskname).configure { onlyIf { false } }
+    }
+  }
+  project.ext.disableTasks = disableMethod
```

Comment:
Why the intermediary variable here? This can be simplified to `ext.disableTasks = { String... taskNames ->`...